chia/wallet/puzzles/notification.clvm

```diff
@@ -0,0 +1,8 @@
+(mod (
+    TARGET
+    MSG_HASH
```

Comment:
why do we need `MSG_HASH` here?

Reply:
I think when I wrote it, I just wanted this to be distinct from a regular ACS puzzle, make something about it "notification" specific but I think you're right, that's probably silly